test/pageobjects/pages.js

```diff
@@ -0,0 +1,9 @@
+
+import Page from "./page.js";
+
+class Pages extends Page {
+    get getTitle () {
+        return $('.mw-page-title-main');
+    }
+}
```

Comment:
I think this `getTitle` can be moved (combined) to `main.page.js` instead of having a separate a file.